client/extensions/woocommerce/lib/analytics/tracks-utils.js

```diff
@@ -20,3 +26,14 @@ export const recordTrack = ( tracks, debug, tracksStore ) => ( eventName, eventP
 
 	tracks.recordEvent( eventName, eventProperties );
 };
+
+export const bumpMCStat = ( debug, tracksStore ) => ( group, name ) => {
+	debug( `stat bump ${ group }: ${ name }` );
+
+	if ( tracksStore.isTestSite() ) {
+		debug( 'stat bump discarded. this site is flagged with `dotcom-store-test-site`' );
+		return { type: WOOCOMMERCE_STAT_DISCARDED, group, name };
```

Comment:
so now with an action being returned we could even have some test coverage around test site stats being discarded correct?

Reply:
Yup with some mocking. Though the import/reliance on the main 'lib/analytics' for the other method makes testing a bit hard because `window` is assumed to exist. Though if we are going to stop relying on our special track function we can clean that up a bit.